BUILD

```diff
@@ -67,6 +67,8 @@ cc_library(
     hdrs = [
         ":gemmlowp_public_headers",
     ],
+    includes = ["."],
+    linkopts = ["-pthread"],
```

Comment:
It is a bit funky I know! I actually based this off the Bazel documentation here:
http://bazel.io/docs/cpp.html
From digging into it a bit more, it seems like it does a few other things besides linking in pthreads, like making errno (ick) thread-local:
http://stackoverflow.com/questions/2127797/gcc-significance-of-pthread-flag-when-compiling